### ISSUE TITLE
Questions about processed data you provide

### ISSUE THREAD

**Issue by gotoHappy** [ForMyCat/SparseGS#9]
Thank you for providing the open-source code and processed data. 
We have observed that the Colmap results of datasets containing different numbers of images appear to share the same coordinate system. How is this achieved?
We are particularly interested in understanding if the presence of sparse-view images might impact the accuracy of camera pose estimation. Upon reviewing the datasets you supplied, we noticed that the extrinsic parameters for the same image are consistent across datasets with varying sampling rates.

**Comment by ForMyCat** [comment]
Hi, we mention in our paper that sparse inputs will degrade the quality of exsisting sfm method (COLMAP), or cause it to fail if scene coverage is insufficient. However, this is not the topic we are exploring in this project. So, instead of running COLMAP with just the 12 input views, we get their camera parameters by parsing from the full model estimated from the full dataset. We then run point_triangulation from the 12 cameras to get the initial pointcloud for training. 

So, long story short, the camera intrinsics and extrinsics are computed and extracted from the same larger dataset, but the initial point cloud is generated using only the perspective training views.

If you want to explore number of views vs. sfm accuracy, I suggest you re-run the entire COLMAP pipeline with the images we provided. We hand picked these images for maximum scene converage so that sfm is more likely to succeed.

**Comment by gotoHappy** [comment]
Thank you for your answer!
